test_runner/flank.yml

```diff
@@ -101,6 +101,24 @@ gcloud:
   # test-targets:
   #  - class com.example.app.ExampleUiTest#testPasses
 
+  ## A map of robo_directives that you can use to customize the behavior of Robo test.
+  ## The type specifies the action type of the directive, which may take on values click, text or ignore.
+  ## If no type is provided, text will be used by default.
+  ## Each key should be the Android resource name of a target UI element and each value should be the text input for that element.
+  ## Values are only permitted for text type elements, so no value should be specified for click and ignore type elements.
+  # robo-directives:
+  # - type: text
+  #   name: input_resource_name
+  #   input: message
+  # - type: click
+  #   name: button_resource_name
+
+  ## The path to a Robo Script JSON file.
+  ## The path may be in the local filesystem or in Google Cloud Storage using gs:// notation.
+  ## You can guide the Robo test to perform specific actions by recording a Robo Script in Android Studio and then specifying this argument.
+  ## Learn more at https://firebase.google.com/docs/test-lab/robo-ux-test#scripting.
+  # robo-script: path_to_robo_script
```

Comment:
Nice work! Remember to update readme.md as well.

Reply:
I've updated bunch of config options in `README.md` and `flank.yml` because both of them had lack of some updates.